apps/meteor/tests/end-to-end/api/chat.ts

```diff
@@ -768,6 +769,44 @@ describe('[Chat]', () => {
 				.end(done);
 		});
 
+		describe('Bad words filter', () => {
+			before(async () => {
+				await updateSetting('Message_AllowBadWordsFilter', true);
+				await updateSetting('Message_BadWordsFilterList', 'badword');
+			});
+
+			after(async () => {
+				await updateSetting('Message_AllowBadWordsFilter', false);
+				await updateSetting('Message_BadWordsFilterList', '');
+			});
+
```

Comment:
Can't we add a test case in which we have more than one badword, to assure it works as well :P